### ISSUE TITLE
Tabbing, keyboard navigation and focusability unavailable when searchable=false

### ISSUE THREAD

**Issue by mofodojodino** [sagalbot/vue-select#97]
In the current implementation of vue-select when you make the component `searchable=false` it uses a `span` instead of the main `input` field. This causes several issues as the `span` element  is not focusable causing tabbing to be disabled and you also lose the standard keyboard navigation that you normally have in searchable mode.

Another side effect of the current implementation is that styling needs to be modified for the non searchable elements to make them look the same.
